site/pages/[...slug].tsx

```diff
@@ -45,31 +44,42 @@ export default function Page({ source, meta, sidebarTree }) {
 
   return (
     <>
-      <NextSeo
-        title={title}
-        description={description}
-        canonical={canonicalUrl}
-        openGraph={{
-          url: canonicalUrl,
-          title,
-          description,
-          type: meta.layout === 'blog' ? 'article' : 'website',
-          article: meta.layout === 'blog' && {
-            publishedTime: meta.date,
-            authors: meta.authorsDetails?.map(a => a.url) || [],
-          },
-          images: [
-            {
-              url: image,
-              alt: title,
+      {meta.layout === 'docs' ?
+        <NextSeo
+          title={title}
+          description={description}
+          canonical={canonicalUrl}
+          openGraph={{
+            url: canonicalUrl,
+            title,
+            description,
+            type: meta.layout === 'blog' ? 'article' : 'website',
```

Comment:
@Datopian-Shreyas this seems to be inside the previous `meta.layout === 'blog'` condition.